incremental-font-transfer/src/glyph_keyed.rs

```diff
@@ -752,6 +752,15 @@ pub(crate) mod tests {
 
         for (tag_a, tag_b) in it_a.zip(it_b) {
             assert_eq!(tag_a, tag_b);
+            if tag_a == Tag::new(b"head") {
+                // zero out checksum_adjustment field
+                let mut a = a.table_data(tag_a).unwrap().as_bytes().to_vec();
+                let mut b = b.table_data(tag_b).unwrap().as_bytes().to_vec();
+                a[8..12].copy_from_slice(&[0, 0, 0, 0]);
```

Comment:
just trivia, but I think you could write this as,
```suggestion
                a[8..12].fill(0);
```